plugins/mcreator-localization/help/fr_FR/entity/attack_knockback.md

```diff
@@ -0,0 +1,2 @@
+The attack knockback defines the knockback value of each attack of this entity

```

Comment:
This one should probably be translated too.

Reply:
Done.